packages/SwingSet/src/kernel/slogger.js

```diff
@@ -74,12 +78,24 @@ function makeFinishersKit(wrappers) {
   });
 }
 
+/** @param {(level: string) => (...args: unknown[]) => void} makeLogger */
+const makeDummyConsole = makeLogger => {
+  const dummyConsole = /** @type {any} */ (
+    Object.fromEntries(logLevels.map(level => [level, makeLogger(level)]))
+  );
+  return /** @type {LimitedConsole} */ (harden(dummyConsole));
+};
+export const badConsole = makeDummyConsole(level => () => {
+  throw Error(`unexpected use of badConsole.${level}`);
+});
+export const noopConsole = makeDummyConsole(_level => () => {});
+
 /**
- * @param {*} slogCallbacks
- * @param {Pick<Console, 'debug'|'log'|'info'|'warn'|'error'>} dummyConsole
+ * @param {SlogWrappers} slogCallbacks
+ * @param {LimitedConsole} [dummyConsole]
  * @returns {KernelSlog}
  */
-export function makeDummySlogger(slogCallbacks, dummyConsole) {
+export function makeDummySlogger(slogCallbacks, dummyConsole = badConsole) {
```

Comment:
Did we consider making `dummyConsole` mandatory ?

Reply:
Not in this refactoring; there's lots of `makeDummySlogger({})` in test code.